`yarn run build` error

## `yarn run build` is giving error

After cloning the react-bootstrap repository, I did yarn `yarn run bootstrap` then I did `yarn run build` to compile the code but it give me following error

```> node tools/build.js

Building targets: all

Building: commonjs modules
Building: es modules
Building: browser distributable
TypeError: execa.shell is not a function
    at shell (/home/mohit/Desktop/TATTLE/main/shell3-in-1/react-bootstrap/tools/build.js:26:28)
    at step (/home/mohit/Desktop/TATTLE/main/shell3-in-1/react-bootstrap/tools/build.js:37:9)
    at /home/mohit/Desktop/TATTLE/main/shell3-in-1/react-bootstrap/tools/build.js:22:9
    at Object.<anonymous> (/home/mohit/Desktop/TATTLE/main/shell3-in-1/react-bootstrap/tools/build.js:80:17)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-bootstrap@1.0.0-beta.9 build: `node tools/build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-bootstrap@1.0.0-beta.9 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:```

--- a/tools/build.js
+++ b/tools/build.js
@@ -23,7 +23,8 @@ const step = (name, root, fn) => async () => {
   console.log(cyan('Built: ') + green(name));
 };
 
-const shell = cmd => execa.shell(cmd, { stdio: ['pipe', 'pipe', 'inherit'] });
+const shell = cmd =>
+  execa(cmd, { stdio: ['pipe', 'pipe', 'inherit'], shell: true });
 
 const has = t => !targets.length || targets.includes(t);
 

